README-dtq.md

```diff
@@ -0,0 +1,146 @@
+
+# DSpace
```

Comment:
why do we copy contents from dspace README?

Reply:
Because for each point from README could be added another information to the README-dtq and I would like to use only our README-dtq. Is that a problem?
Another solution is to keep additional information only in the README-dtq but that will be quite discomfort for me because I will must to check two README files.